grakn-graql-shell/src/main/java/ai/grakn/graql/GraqlShell.java

```diff
@@ -262,10 +272,12 @@ private static String loadQuery(String filePath) throws IOException {
             return lines.stream().collect(joining("\n"));
     }
 
-    private static void sendBatchRequest(String uriString, String graqlPath, String keyspace) throws IOException {
+    private static void sendBatchRequest(LoaderClient loaderClient, String graqlPath, String activeTasks, String batchSize) throws IOException {
         AtomicInteger numberBatchesCompleted = new AtomicInteger(0);
 
-        LoaderClient loaderClient = new LoaderClient(keyspace, uriString).setRetryPolicy(true);
+        if (Integer.parseInt(activeTasks)!=0) loaderClient.setNumberActiveTasks(Integer.parseInt(activeTasks));
+        if (Integer.parseInt(activeTasks)!=0) loaderClient.setBatchSize(Integer.parseInt(batchSize));
```

Comment:
We should consider the case when a user passes something that is not an integer:
`graql.sh -b file.gql -a "users are the worst"`

I would parse and handle the edge cases outside of `sendBatchRequest`, then just passing the integers into this method.